tbcrawler/pytbcrawler.py

```diff
@@ -55,6 +57,11 @@ def run():
     job_config = ut.get_dict_subconfig(config, args.config, "job")
     job = crawler_mod.CrawlJob(job_config, url_list)
 
+    # Run display
+    xvfb_wrapper = XvfbWrapper()
+    xvfb_wrapper._from(args.virtual_display)
+    xvfb_wrapper.start()
+
```

Comment:
Very educative talk on the subject:
https://www.youtube.com/watch?v=o9pEzgHorH0